roles/container-engine/cri-o/tasks/crio_repo.yml

```diff
@@ -149,6 +149,13 @@
     - repo: "fedora-modular"
       section: "fedora-modular"
 
+- name: Enable CRI-O ex module
+  command: "rpm-ostree ex module enable cri-o:{{ crio_version }}"
+  become: true
+  when:
+    - is_ostree
+    - ostree_version is version('2021.9', '>=')
```

Comment:
Unless `ostree_version` is in `detaults/main.yml` this would fail.

Reply:
That's because I assume if `is_ostree` is defined then `ostree_version` will be defined too.